docs/scos/dev/front-end-development/202212.0/oryx/oryx-versioning.md

```diff
@@ -72,3 +72,30 @@ Minor releases are fully backward compatible and do not require any developer as
 ### Deprecation practices
 
 If a newer and better alternative to a feature is introduced in a minor release, the old one is deprecated. Deprecated features remain functional until the next major release, in which they are removed completely.
+
+## Feature Flag Versioning
+
+### Overview
+
+Oryx introduces Feature Flag Versioning to add new functionalities without affecting backward compatibility. This is achieved using the `ORYX_FEATURE_VERSION` environment variable.
+
+### Code Optimization
+
+Setting `ORYX_FEATURE_VERSION` optimizes your build by leveraging Dead Code Elimination (DCE). Any code unrelated to the specified version will not be included in the final bundle, making your application leaner.
```

Comment:
it's cool, but not sure if we need to add this detail to the docs here

Reply:
I put it here not because it's cool, but if you won't use it, DCE won't work, so good to know to have optimal bundle. But maybe I can rephrase?